### PR TITLE
Fix unit for hw.network.bandwidth.limit metric

### LINKED ISSUE #97: Units for `hw.network.bandwidth.limit` metric
**What are you trying to achieve?**

In the `hw.network` instrument, the `hw.network.bandwidth.limit` metric is defined as:
* Description: Link speed
* Units: By
* Data point type: UpDownCounter

The units are currently set to "By" (bytes), which is independent of time. However, the bandwidth is the capacity of a link to transmit/receive data, expressed as a quantity of information (e.g., bits or bytes) per unit of time (e.g. second).

What did you expect to see?

I was expecting units for the `hw.network.bandwidth.limit` metric to be something like `By/s`.

**Additional context.**

Add any other context about the problem here. If you followed an existing documentation, please share the link to it.

### PR DESCRIPTION
**Pull request by bertysentry**
Unit for `hw.network.bandwidth.limit` must be `By/s`

Fixes #97.

### CODE REVIEW

**Comment by bertysentry** [comment]
@jsuereth Link check failed because there are broken link in another Markdown file (links to IBM documentation).

**Review by arminru — APPROVED** [review]

**Comment by arminru** [comment]
> @jsuereth Link check failed because there are broken link in another Markdown file (links to IBM documentation).

@bertysentry This should be fixed now with #92

File: specification/metrics/semantic_conventions/hardware-metrics.md
Comment (sebastien-rosset) [reply specification/metrics/semantic_conventions/hardware-metrics.md:273]: I agree with you it's odd to think of the link speed as a UpDownCounter that can be incremented/decremented. The operational link speed is whatever is negotiated by both ends of the network link.

The link speeds can be added across multiple ports in a network device. For example, a switch may have 32 physical ports, 16 of them are configured, and at any time the operational state may change depending on the link status.

Comment (bertysentry) [reply specification/metrics/semantic_conventions/hardware-metrics.md:273]: The use of `UpDownCounter`  for `*.limit` metrics has been discussed in this [this PR](https://github.com/open-telemetry/opentelemetry-specification/pull/2518). I understand it's a bit counter-intuitive (I was fighting against this myself until I understood what it really meant 😅). The key is: It's a summable metric (calculating the sum of bandwidth for all network interfaces of a system is a valid use case, unlike summing up bandwidth utilization values). That's why `UpDownCounter` is the proper metric type for `hw.network.bandwidth.limit`.

Also, the available bandwidth of a network interface can change over time depending on the link negotiation. If you monitor your Wifi adapter, you'll see its speed changing constantly, sometimes several times per minute. This is one more reason for using `UpDownCounter`.

Comment (joaopgrassi) [reply specification/metrics/semantic_conventions/hardware-metrics.md:273]: Perhaps then we need to better document these to clarify any future confusion? When we convert the markdown to YAML to generate these, then we can use `note`, but for now it can be just a foot note or something. What do you think?

Comment (sebastien-rosset) [reply specification/metrics/semantic_conventions/hardware-metrics.md:273]: BTW, the "link speed" is ambiguous. It could be the configured link speed or the operational link speed. Some network interfaces have both a concept of configured and operational link speed.

Comment (sebastien-rosset) [reply specification/metrics/semantic_conventions/hardware-metrics.md:273]: Just my 2 cents, opening a separate PR seems a better option, otherwise this simple fix may end up being delayed.

Comment (sebastien-rosset) [reply specification/metrics/semantic_conventions/hardware-metrics.md:273]: How does one determine whether a metric should be considered additive or not? In most cases, it's obvious. But what about `hw.power` which is a gauge. Yes, it does not make sense to sum `hw.power` measurements taken at different timestamps, but it may be useful to sum `hw.power` measurements from multiple devices at a given timestamp. For example, that could be used to analyze the power drawn versus the circuit capacity.

Comment (joaopgrassi) [reply specification/metrics/semantic_conventions/hardware-metrics.md:273]: @sebastien-rosset  

> How does one determine whether a metric should be considered additive or not

The spec has some guidelines https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/supplementary-guidelines.md#guidelines-for-instrumentation-library-authors

> may be useful to sum hw.power measurements from multiple devices at a given timestamp
You can maybe achieve this by summing them up in the backend? I didn't read them in full so I might be off here.

**Review by joaopgrassi — APPROVED** [review]

**Comment by joaopgrassi** [comment]
Ping @open-telemetry/specs-semconv-approvers @open-telemetry/specs-semconv-maintainers. This should be good to merge, I will left it open for a day then will merge if no one has any objections.

**Comment by bertysentry** [comment]
lol, is this going to be ever merged? 😅

**Comment by bertysentry** [comment]
Thank you @reyang 😊

**Pull request merged**
